apps/webcomponents/src/app/webcomponents.module.ts

```diff
@@ -100,7 +103,7 @@ const CUSTOM_ELEMENTS: [new (...args) => BaseComponent, string][] = [
     },
   ],
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
-  // bootstrap: [AppComponent],
+  bootstrap: [AppComponent],
```

Comment:
This is only for dev purposes I think and should be commented again.

Reply:
I still see this uncommented

Reply:
Yes indeed (that's because I have to uncomment it to test, cf my other answer :slightly_smiling_face: You should also uncomment it to test)

Reply:
I think the web component should also work with this commented. It is just to make the web component work like an angular component for dev if I remember right.

Reply:
Do not forget to comment this before merging.